tests/test_response_processor.py

```diff
@@ -4,132 +4,108 @@
 from app import settings
 from unittest.mock import MagicMock
 from structlog import wrap_logger
-from app.response_processor import ResponseProcessor, get_ftp_folder
-from tests.test_data import survey_no_heartbeat, survey_heartbeat_true
+from app.response_processor import ResponseProcessor, get_ftp_folder, is_census
+from tests.test_data import survey_census, survey_023
 
 logger = wrap_logger(logging.getLogger(__name__))
 
 
 class TestResponseProcessor(unittest.TestCase):
-    RESPONSE_WITHOUT_TX = '''{
-            "metadata": {
-              "user_id": "789473423",
-              "ru_ref": "12345678901A"
-            }
-        }'''
-
-    RESPONSE_WITH_TX = '''{
-            "tx_id": "0f534ffc-9442-414c-b39f-a756b4adc6cb",
-            "metadata": {
-              "user_id": "789473423",
-              "ru_ref": "12345678901A"
-            }
-        }'''
-
-    XML_RESPONSE = '''{
-            "file-type": "xml",
-            "tx_id": "0f534ffc-9442-414c-b39f-a756b4adc6cb",
-            "metadata": {
-              "user_id": "789473423",
-              "ru_ref": "12345678901A"
-            }
-        }'''
+
+    def setUp(self):
+        self.survey = json.loads(survey_023)
+        self.census = json.loads(survey_census)
```

Comment:
I don't think it's a good idea for the tests to share the test data in this way. A test can (and further on does) change the test data for subsequent tests. This is unclear and could lead to tests breaking in odd ways (for example if someone modified the order of the tests). I think it was clearer before with each test loading it's test data.